tests/test_sunrise.py

```diff
@@ -1,8 +1,371 @@
-from sunrise import frac_year
+from sunrise import *
```

Comment:
```suggestion
from sunrise import *
from pytest import approx
```
We need to import approx from pytest